internal/server/server.go

```diff
@@ -443,6 +444,7 @@ func (s *Server) mkGRPCServer(log *zap.Logger, auditLog audit.Log) (*grpc.Server
 		grpc.ChainStreamInterceptor(
 			grpc_recovery.StreamServerInterceptor(),
 			telemetryInt.StreamServerInterceptor(),
+			otelgrpc.StreamServerInterceptor(),
```

Comment:
It turns out that the approach of using the client interceptors doesn't work out of the box, because we only read the tracing headers from the HTTP request, whereas the client injects them into the gRPC request metadata.